CONTRIBUTING.md

```diff
@@ -101,7 +104,7 @@ Go to your Jira instance that you created earlier and do the following steps:
 
 ### Setting up the App
 
-In your Jira instance, in the `Manage Apps` section, click on your App's button, then click on `Get Started`.  This will bring you to the App's dashboard.  Click the `Add an Organization` button and follow the steps to install the App on Github and allow it permission to view your repos.
+In your Jira instance, in the `Manage Apps` section, click on your App's button, then click on `Get Started`.  This will bring you to the App's dashboard.  Click the `Connect GitHub organisation` button and follow the steps to install the App on GitHub and allow it permission to view your repos.
```

Comment:
nit: z instead of s for organization

Reply:
ewwwww :)

Reply:
haha I know, but just to keep the pattern from https://github.com/atlassian/github-for-jira/blob/main/views/jira-configuration.hbs#L24

Reply:
Yea I think Jira also uses 'z' so makes sense